app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills += "JavaScript", "HTML", "CSS"
```

Comment:
maybe here would be better to be not "JavaScript", "HTML", "CSS", but specifically list

Reply:
I agree, then it won't confuse anybody and will be more readable